lib/perl/Genome/Disk/Command/Volume/SyncUsage.pm

```diff
@@ -27,44 +25,30 @@ class Genome::Disk::Command::Volume::SyncUsage {
             doc => 'Sync unallocated_kb?',
         },
     },
-    has_transient => {
-        tie_stderr => {
-            is => 'Boolean',
-            default => 1,
-            doc => '(warning) globally tie STDERR to this logger',
-        },
-    },
-    doc => 'Sync usage info for volume (e.g. total KB and unallocated KB)',
+    doc => 'Sync total and unallocated KB usage for volumes',
 };
 
 sub help_detail {
-    'Sync usage info for volume (e.g. total KB and unallocated KB).'
+    return __PACKAGE__->__meta__->doc;
 }
 
 sub execute {
     my $self = shift;
+    $self->status_message('Disk volume sync usage...');
 
-    my %args = (verbose => 1);
     for my $volume ( $self->volumes ) {
-        my $transaction = UR::Context::Transaction->begin;
-        try {
-            $self->info('Syncing volume: '.$volume->mount_path);
-            if ($self->total_kb)      {
-                $self->debug('Sync total kB...');
-                $volume->sync_total_kb(%args);
-            }
-            if ($self->unallocated_kb) {
-                $self->debug('Sync unallocated kB...');
-                $volume->sync_unallocated_kb(%args);
-            }
-            $transaction->commit or die 'Failed to commit volume! '.$volume->mount_path;
+        $self->status_message('Syncing volume: %s', $volume->mount_path);
+        for my $type (qw/ total unallocated /) {
```

Comment:
It might make sense if you think `df` is expensive and you trust your volumes not to change sizes.  I don't trust our volumes not to change size, though! 

(It's fine by me if you want to leave it as-is, or if you want to take those options out.)